build.gradle.kts

```diff
@@ -9,6 +9,7 @@ plugins {
     alias(libs.plugins.android.application) apply false
     alias(libs.plugins.kotlin) apply false
     alias(libs.plugins.kotlinter) apply false
+    id("org.jetbrains.dokka") version "1.9.20"
```

Comment:
This doesn't appear to function correctly. While the plugin is successfully loaded, running `./gradlew dokkaHtml` doesn't actually generate the documentation. I believe this line needs to be moved to `TeamCode/build.gradle`.

Also usually I'd like to specify plugins [in the version catalog](https://developer.android.com/build/migrate-to-catalogs), but I won't block on it.